src/actions.tsx

```diff
@@ -0,0 +1,41 @@
+import * as constants from './constants';
```

Comment:
You'll probably eventually have multiple actions files once things grow in complexity. Just need to think about naming and structure - same comment about the reducers

Reply:
The currently scoped UI has rather simple actions so we kept constants in a single file. We can refactor if the complexity of the interactions grows.